lib/Logger/LogAppenderFile.cpp

```diff
@@ -170,6 +171,8 @@ LogAppenderFile::LogAppenderFile(std::string const& filename, std::string const&
   _useColors = ((isatty(_fd) == 1) && Logger::getUseColor());
 }
 
+LogAppenderFile::~LogAppenderFile() { clear(); }
```

Comment:
I am a bit worried about this. We can have multiple `LogAppenderFile` objects, for example, if we log to different output files (which is allowed and up to the user). Shutting down one of the `LogAppenderFile` objects would then close all of them, which probably isn't desired.